oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Configuration.java

```diff
@@ -281,4 +282,10 @@
             name = "Document Node Store Compression",
             description = "Select compressor type for collections. 'Snappy' is the default supported compression.")
     String collectionCompressionType() default "snappy";
+
+    @AttributeDefinition(
+            name = "Suspend timeout",
+            description = "Timeout for a suspended commit after it conflicted with" +
+                    "a change that is not yet visible. Default: 60000 (milliseconds).")
```

Comment:
this hardcodes the default to 60000 in osgi metadata - could maybe also refer to DEFAULT_SUSPEND_TIMEOUT?